quixstreams/state/rocksdb/windowed/transaction.py

```diff
@@ -240,3 +218,46 @@ def get_windows(
                 result.append(((start, end), value))
 
         return result
+
+    def _set_latest_timestamp(self, prefix: bytes, timestamp_ms: int):
+        self._latest_timestamps[prefix] = timestamp_ms
+        self.set(
+            key=LATEST_TIMESTAMP_KEY,
+            value=timestamp_ms,
+            prefix=prefix,
+            cf_name=LATEST_TIMESTAMPS_CF_NAME,
+        )
+
+    def _get_last_expired_timestamp(self, prefix: bytes) -> Optional[int]:
+        cached_ts = self._last_expired_timestamps.get(prefix)
+        if cached_ts is not None:
+            return cached_ts
+
+        stored_ts = self.get(
+            key=LATEST_EXPIRED_WINDOW_TIMESTAMP_KEY,
+            prefix=prefix,
+            cf_name=LATEST_EXPIRED_WINDOW_CF_NAME,
+        )
+        self._last_expired_timestamps[prefix] = stored_ts
+        return stored_ts
```

Comment:
We don't want to return 0 here like for latest_timestamp?

Reply:
Nope, not in this case.
The previous code expected `None`, so I don't want to change it.